analogin.py

```diff
@@ -138,6 +142,18 @@ def init(self):
                     self.start_trigger.cfg_dig_edge_start_trig(
                         trigger_source = self.startTrigger.source,
                         trigger_edge=self.startTrigger.edge)
+            
+            except DaqError as e:
+                # end the task nicely
+                self.stop()
+                self.close()
+                msg = '\n AnalogInput task initialization failed'
```

Comment:
Do we want to preserve original error message? eg write:
`msg = f"{e}\n AnalogInput task initialization failed"`
instead.

Reply:
yes, that is preferable. I'll go through and add that in

Reply:
I don't think it is necessary. The trace-back should make it clear what errors were raised to eventually obtain the hardware error so we don't need to manually do that job.

Reply:
@c-poole that was my original intent for how this would function. sounds good, i'll leave it for now